doc/ref/groups.xml

```diff
@@ -481,6 +481,7 @@ the series without destroying the properties of the series.
 <#Include Label="ContainedConjugates">
 <#Include Label="ContainingConjugates">
 <#Include Label="MinimalFaithfulPermutationDegree">
+<#Include Label="MinimalFaithfulPermutationDegreeOfSimpleGroup">
```

Comment:
But if you install this as a method for `MinimalFaithfulPermutationDegree`, there isn't much of a reason to document `MinimalFaithfulPermutationDegreeOfSimpleGroup`, is there? It might be different if it was a helper function that takes a description of the simple group (e.g. the `info` record) -- then it could be used in other algorithms (without forcing them to create a group just in order to be able to access the data provided by your function).

Of course the documentation for `MinimalFaithfulPermutationDegree` could be extended to mention that there is special code for simple groups, but I am not sure that this is useful for regular users?